### PR TITLE
Fix:Removing unused fields from MainActivity.java

### LINKED ISSUE #1585: Removing unsued fields in MainActivity.java
**Actual Behaviour**

There are some unused fields in MainActivity.java

**Expected Behaviour**

It should not contain them.

**Steps to reproduce it**

Remove the fields.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by riyasingh1004**
Fixes #1585 

**Changes**:

Removed unused field in the main activity file.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 

[MainActivity.zip](https://github.com/fossasia/pslab-android/files/2937932/MainActivity.zip)

### CODE REVIEW

**Comment by riyasingh1004** [comment]
@cweitat  kindly review

File: app/src/main/java/io/pslab/activity/MainActivity.java [hunk 1]
```diff
@@ -180,7 +180,7 @@ public void run() {
         }
     }
 
-    private Fragment getHomeFragment() throws IOException {
+    private Fragment getHomeFragment() throws IOException{
```
Comment (harsh-2711) [inline app/src/main/java/io/pslab/activity/MainActivity.java]: @riyasingh1004 Do we need this change?

Comment (riyasingh1004) [reply app/src/main/java/io/pslab/activity/MainActivity.java]: I was asked to include this change by @cweitat

Comment (harsh-2711) [reply app/src/main/java/io/pslab/activity/MainActivity.java]: He told you to return the spacing i.e. keep the spacing as it is. Don't change the lines not needed in this PR

Comment (riyasingh1004) [reply app/src/main/java/io/pslab/activity/MainActivity.java]: Oh okay will do so. Sorry for misunderstanding.

**Pull request closed without merging**